ui/shared/BugFiler.jsx

```diff
@@ -159,22 +159,35 @@ export class BugFilerClass extends React.Component {
     }
 
     const jg = jobGroupName.toLowerCase();
-    if (jg.includes('xpcshell') || jg.includes('mochitest')) {
+    if (
+      jg.includes('xpcshell') ||
+      jg.includes('mochitest') ||
+      jg.includes('web platform tests')
+    ) {
       // simple hack to make sure we have a testcase in the summary
-      const isTestPath = [
+      let isTestPath = [
         /.*test_.*.js/, // xpcshell
         /.*test_.*.html/, // mochitest
         /.*test_.*.xhtml/, // mochitest-chrome
         /.*browser_.*.html/, // b-c
         /.*browser_.*.js/, // b-c
       ].some((regexp) => regexp.test(summaryString));
 
+      if (jg.includes('web platform tests')) {
+        isTestPath = [/.*.js \|/, /.*.html \|/, /.*.htm \|/].some((regexp) =>
```

Comment:
dots should be backslashed, not surre if we have `.xhtml` or similar files here.

Reply:
added that in, good catch.